dali/test/python/test_external_source_dali.py

```diff
@@ -397,10 +397,11 @@ def test_ndim_changing():
 
 @raises(RuntimeError, glob="Expected data with layout: \"H\" and got: \"W\"")
 def test_layout_data_mismatch():
-    src_pipe = Pipeline(1, 1, 0)
+    src_pipe = Pipeline(1, 1, 0, prefetch_queue_depth=1)
     src_pipe.set_outputs(fn.external_source(name="input", layout="H"))
     src_pipe.build()
     src_pipe.feed_input("input", [np.zeros((1))], layout="W")
+    src_pipe.run()
```

Comment:
Since `ValidateInputs` are now in `SetupImpl`, not `SetDataSource` (so a step later), this test had to be updated.